### ISSUE TITLE
filter by alpha3

### ISSUE THREAD

**Issue by missinglink** [pelias/api#44]
it might be nice to allow consumers to restrict the results returned to only those which lie within an alpha3.

ie. I want an autocomplete for admin records in GBR.

**Comment by tpievila** [comment]
The bug in elasticsearch has been closed, so is this feature ready to be merged back again?

**Comment by missinglink** [comment]
hi @tpievila, unfortunately the bug was closed as a 'wontfix' in favour of https://github.com/elastic/elasticsearch/issues/8909 which will introduce the new 'Completion Suggester Version 2' in elasticsearch version `2.0`.

I am pretty confident that this issue will be resolved once `2.0` lands.

**Comment by tpievila** [comment]
On Wed, Mar 11, 2015 at 10:39:43AM -0700, Peter Johnson a.k.a. insertcoffee wrote:
> hi @tpievila, unfortunately the bug was closed as a 'wontfix' in favour of https://github.com/elastic/elasticsearch/issues/8909 which will introduce the new 'Completion Suggester Version 2' in elasticsearch version `2.0`.
> 
> I am pretty confident that this issue will be resolved once `2.0` lands.

Ah... mea culpa for reading the issue thread hurriedly.

-- 
Tomi Pievil�inen, +358 400 487 504
A: Because it disrupts the natural way of thinking.
Q: Why is top posting frowned upon?

**Comment by dianashk** [comment]
Duplicate pelias/api#112
